tests/files/const22.gno

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+)
+
+const (
+	ia = 1_000_000
+	ib = 1_00_00_00
+	ic = 1_000
+
+	fa = 1_000_000.000
+	fb = 1_000_000.000_000
+)
```

Comment:
I'd suggest adding some more just to make sure we cover all cases:

- `100_000i` (imag number)
- `1+100_000i` (complex)
- `0x_16_32` (hex)
- `0b_0110_1001` (bin)
- `0o_644_755` (oct)